src/utils/utils.js

```diff
@@ -70,7 +70,7 @@ export function navigateToDocumentsList(path, projectId, userId) {
     slash = "";
   }
 
-  const parameters = `?project=${projectId}&is_reviewed__exact=0&assignee__id__exact=${userId}`;
+  const parameters = `?project=${projectId}`;
```

Comment:
If we no longer need `userId` for this redirect, it should be removed from the parameters of `navigateToDocumentsList`, and calls to that function (I see only one in `DocumentEdit.vue`) should be updated accordingly.